packages/espresso/example/android/app/build.gradle

```diff
@@ -62,28 +62,28 @@ dependencies {
     implementation "androidx.multidex:multidex:2.0.1"
 
     // Core library
-    api 'androidx.test:core:1.2.0'
+    api 'androidx.test:core:1.6.1'
```

Comment:
IIRC this is `api` because it's actually re-exposed as part of the public (native) API surface, so if anything here is breaking we'd want to version accordingly (although since it's a minor version change, maybe it's not breaking?)

Reply:
This value has not change in 5 years since the creation of this plugin. 
I am going to assume it is the most breaking change we can make.

I will update this to have more obviously breaking version number once I get the tests passing. 
https://github.com/flutter/packages/commit/dcace54ce86539c9535c4890759dd8a538624fbc#diff-af20d9c2b386a0c999752ae955c2295514c30f75e7aa6255a47e2c6114340ef9